.github/workflows/build.yml

```diff
@@ -365,7 +365,23 @@ jobs:
         with:
           node-version: ${{ env.NODEJS_VERSION }}
 
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODEJS_VERSION }}
+
+      - name: Tag commit
+        uses: tvdias/github-tagger@v0.0.1
+        with:
+          repo-token: "${{ secrets.PROJEN_GITHUB_TOKEN }}"
+          tag: "v${{ needs.prepare.outputs.version }}"
```

Comment:
Is this step idempotent? If not can we make it idempotent?

Reply:
It's not, I can see if I can get it to be though

Reply:
done (hopefully)